src/packet/enums.rs

```diff
@@ -82,12 +96,15 @@ impl EnumConverter for PacketType {
     fn to_u8(&self) -> u8 {
```

Comment:
maybe we can have `Into` trait implemented here instead, just a thought for later PR.